### ISSUE TITLE
parameters imported from petab with estimate=False become "super_fixed"

### ISSUE THREAD

**Issue by FFroehlich** [ICB-DCM/pyPESTO#217]
As in, they are hardcoded in `objective.mapping_par_opt_to_par_sim` and changes to input value in `x` when calling the objective are ignored.

**Comment by yannikschaelte** [comment]
yes, that is right because we assume that all changes to the parameters happen in the table itself. of course, it could be good if values in the x_fixed vector can override those values (I assume that's what you mean and that's what's currently not happening)?

**Comment by FFroehlich** [comment]
Pypesto creates a fixed_parameter for that value, so I expect that it behaves the same way as if I manually created a fixed_parameter in pypesto. If there just wouldn’t be a fixed_parameter in pypesto, I would be unhappy because that feature doesnt exist. In the current implementation I can pass values for these parameters in the objective function and they don’t have any effect. I consider that a bug.
